scripts/x-data-grid-premium.exports.json

```diff
@@ -451,6 +452,7 @@
   { "name": "GridPanelWrapper", "kind": "Variable" },
   { "name": "GridPanelWrapperProps", "kind": "Interface" },
   { "name": "GridParamsApi", "kind": "Interface" },
+  { "name": "GridParamsPrivateApi", "kind": "Interface" },
```

Comment:
Probably shouldn't be exported publically.

Reply:
Is there a convention to prevent it from being exported yet have it co-located with the public interface? Not entirely sure what this file does to be honest (haven't dug into it), but also saw `GridRowProPrivateApi` as a part of it, which doesn't show up in any docs.

Reply:
AFAIU, this is auto-generated from the exports available at `@mui/x-data-grid{,-pro,-premium}`, so anything exported from the root `index.ts(x)` ends up here.

> Is there a convention to prevent it from being exported yet have it co-located with the public interface? 

No, somewhat annoying but if there is an `export * from '...'` that ends up in the root `index.ts(x)`, it needs to be replaced with `export { a, b, c } from '...'`.

Reply:
Ok, but that's good enough. I was afraid it scans models directory or something. Fixed that.